mmv1/third_party/terraform/.teamcity/components/get_services.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+import generated.ServicesListGa
+import generated.ServicesListBeta
+
+// This file is maintained in the GoogleCloudPlatform/magic-modules repository and copied into the downstream provider repositories. Any changes to this file in the downstream will be overwritten.
+
+fun getServicesList(Services: Array<String>, version: String): Map<String,Map<String,String>> {
```

Comment:
Could you please add a comment about what the function is used for? Looks like the comment should say that it's used to create a list of a subset of services relevant to testing a feature branch, so only relevant builds are made.

Also, given it's so coupled to feature branch testing, what do you think about moving it into the feature_branches folder?

Reply:
thought I had added a comment explaining the usage, I'll include it. 

I agree it makes sense since it's intended more for feature branches.